runtime/stdlib/flow.go

```diff
@@ -271,3 +271,12 @@ var AccountInboxClaimedEventType = newFlowEventType(
 	AccountEventRecipientParameter,
 	AccountEventNameParameter,
 )
+
+var AccountLinkedEventType = newFlowEventType(
+	"AccountLinked",
+	AccountEventAddressParameter,
+	sema.Parameter{
+		Identifier:     "path",
+		TypeAnnotation: sema.NewTypeAnnotation(sema.PrivatePathType),
```

Comment:
Is there a way to ensure the type used here is the same as the param type in the [actual account link function](https://github.com/onflow/cadence/blob/1775ebd3bfa90a29653c16d99d4a0253ee86f3b7/runtime/sema/authaccount_type.go#L83)? Just to be sure that we won't update one and forget to update the other.

Reply:
That's a great idea! Refactored to a common value in 7c092a8a04e132a12b6fb299f31446120bb3c98a